renovate.json5

```diff
@@ -33,6 +33,14 @@
       "matchUpdateTypes": ["patch", "minor"],
       "groupSlug": "all-minor-patch",
     },
+    // The testsuite depends directly on the tests in `graphql-http`. We want
+    // integrations using our testsuite to have a deterministic set of
+    // tests run so we should pin it. This way, the tests being run don't
+    // surprisingly change in the integration repo with, e.g a lockfile update.
```

Comment:
Maybe be even more explicit like "Because of this, updates to `graphql-http` affect the contents of published packages, require a changeset, and cannot be auto-merged. So we split it into its own Renovate PRs so that it doesn't block other upgrades."

(And once it's in its own group we could imagine adding something to CI or a renovate plugin or something that auto-creates the changeset...)

Reply:
Good feedback, I pulled it in to my comment.